.travis.yml

```diff
@@ -1,10 +1,18 @@
-language: node_js
-node_js: '0.10'
-services: elasticsearch
-sudo: true
+sudo: required
+
+services:
+  - docker
+
 branches:
   only:
   - master
-before_install:
-  - ./build/setup_ubuntu.sh
+
+install: true
+
+script:
+  - docker build --rm --tag=hmda/loader .
+  - CONTAINER_ID=$(docker run -d -p 9200:9200 -p 9300:9300 elasticsearch:1.7.3)
   - sleep 10
+  - ES_HOST=$(docker inspect "$CONTAINER_ID" | grep IPAddress | cut -d '"' -f 4)
+  - echo "$ES_HOST"
```

Comment:
Yeah, I kinda purposely left this just because I thought it could be helpful... One line of output doesn't hurt too much